### PR TITLE
fix(katana): make sure value is within FieldElement range

### PR DESCRIPTION
**Pull request by lambda-0x**
Fix: #845 

Doing mod by `0x9` caused some randomly generated values to be out of range. so mod by `0x8` to make sure any randomly generated value is within range.

### CODE REVIEW

**Comment by lambda-0x** [comment]
(is there any place where we hard coded this private keys, because of this change we need to update those)

**Comment by lambda-0x** [comment]
(is there any other place where we hard coded this private keys? because of this change we need to update those)

File: examples/ecs/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Dojo ECS Example
 
-This repo contains a simple example of using the Dojo ECS system. It defines a simple game where a player has a limited set of moves and can move their position through executing the move system.
+This repo contains a simple example of using the Dojo ECS system. It defines a
+simple game where a player has a limited set of moves and can move their
+position through executing the move system.
```
Comment (lambda-0x) [inline examples/ecs/README.md:5]: this was auto formatted by vscode, lmk if this is a problem i will remove formatting related stuff from this PR.

**Review by kariy — COMMENTED** [review]
lgtm. i believe `0x8` is the right value to limit the msb

need to fix the readme a bit.

can you also change the values in the example [`Scarb.toml`](https://github.com/dojoengine/dojo/blob/857cbc3db064fed11cf74aaaf53b8e3e3858ed21/examples/ecs/Scarb.toml)

File: examples/ecs/README.md (hunk 1 above)
Comment (lambda-0x) [reply examples/ecs/README.md:5]: done

**Comment by lambda-0x** [comment]
@kariy done, found few more places where old address was used (missed them because they started with `0x03e` instead of `0x3e` so do review that once again.

**Comment by kariy** [comment]
> @kariy done, found few more places where old address was used (missed them because they started with `0x03e` instead of `0x3e` so do review that once again.

it seems the private keys changed slightly, the first account private key is now `0x18000000003000001800000000000300000000000030060018000066001`

can you fix that as well? thanks

File: crates/dojo-erc/Scarb.toml [hunk 2]
```diff
@@ -24,6 +24,6 @@ test = "sozo test"
 [tool.dojo.env]
 # Katana
 rpc_url = "http://localhost:5050"
-account_address = "0x03ee9e18edc71a6df30ac3aca2e0b02a198fbce19b7480a63a0d71cbd76652e0"
+account_address = "0x517ececd29116499f4a1b64b094da79ba08dfd54a3edaa316134c41f8160973"
 private_key = "0x0300001800000000300000180000000000030000000000003006001800006600"
```
Comment (kariy) [inline crates/dojo-erc/Scarb.toml:28]: here

File: examples/ecs/Scarb.toml [hunk 3]
```diff
@@ -31,5 +31,5 @@ initializer_class_hash = "0xbeef"
 rpc_url = "http://localhost:5050/"
 
 # Default account for katana with seed = 0
-account_address = "0x03ee9e18edc71a6df30ac3aca2e0b02a198fbce19b7480a63a0d71cbd76652e0"
+account_address = "0x517ececd29116499f4a1b64b094da79ba08dfd54a3edaa316134c41f8160973"
 private_key = "0x0300001800000000300000180000000000030000000000003006001800006600"
```
Comment (kariy) [inline examples/ecs/Scarb.toml:35]: also here

**Comment by lambda-0x** [comment]
oops

**Pull request merged**